### ISSUE TITLE
6to5

### ISSUE THREAD

**Issue by amcdnl** [gocardless/es6-angularjs#5]
Would love to see 6to5 here instead of tracuer since its becoming the 'standard'.

**Comment by jackfranklin** [comment]
We do plan to look into 6to5 as I believe soon it will be supported in SystemJS - and once we do look at it and if we decide to go ahead with it we will update this :) If you do it yourself please do let us know how it goes - would be interested to hear. We like the idea of 6to5.

**Comment by amcdnl** [comment]
I'm researching it right now.  A guy on gitter sent me this, he's using it here : https://github.com/aurelia/dependency-injection

have yet to get it included in my proj though.

**Comment by amcdnl** [comment]
@jackfranklin I noticed u guys also have the https://github.com/gocardless/systemjs-assetgraph proj too.  Are you lazy loading angular modules w/ that?

**Comment by jackfranklin** [comment]
Not yet. We are keen and have on our backlog a task to explore lazy loading, but it's not something we do yet. Again, on the roadmap.

**Comment by amcdnl** [comment]
@jackfranklin right on.  its not easy :S ... I've got lazy loading working right now in Require using https://github.com/ocombe/ocLazyLoad but migrating towards System so just curious if others have crossed the bridge.

FYI - when i did a build I got an error at the last - http://www.screencast.com/t/iJGnnroi3X6G

One other note, I was wondering why you did the import approach for templates like: `import template from './auth-required.template.html!text';` vs doing a html2js ( https://github.com/karlgoldstein/grunt-html2js ) and injecting them on build time.

**Comment by amcdnl** [comment]
@jackfranklin check out my project, I've got 6to5 working with system and angular and some other fancy stuff too.

https://github.com/amcdnl/systemjs-test

**Comment by jackfranklin** [comment]
Cool, we'll be sure to check it out, thanks! + @harrison

**Comment by guybedford** [comment]
@amcdnl awesome! I've set up a community page on the SystemJS repo, and add this with the GoCardless stuff as well - https://github.com/systemjs/systemjs/wiki/Community-Projects.

**Comment by feelepxyz** [comment]
Awesome! I'm really keen on lazy loading our app, will check it out.

**Comment by amcdnl** [comment]
@harrison ya, that was super tricky to get that worked out just right.  I had a different approach before where I defined the routes and then would lazy load the script with require and inject the modules but it was pretty hacky and things could get out of order.  I'm working with the guy on the future state proj to get his code updated since I forked some of his code there.  My build process also does bundling 'on-the-fly', its not perfect yet but it works on that project; any feedback would be appreciated.

@guybedford thanks!  only thing I have left is to get the cache-busting to work right with plugins.
